dotnetv3/dynamodb/scenarios/DynamoDB_Basics/DynamoDB_Basics_Scenario/DynamoDbMethods.cs

```diff
@@ -0,0 +1,432 @@
+                TableName = tableName,
+            };
+
+            var response = await client.UpdateItemAsync(request);
+
+            return response.HttpStatusCode == System.Net.HttpStatusCode.OK;
+        }
+
+        // snippet-end:[DynamoDB.dotnetv3.UpdateItem]
+
+        // snippet-start:[DynamoDB.dotnetv3.GetItem]
+
+        /// <summary>
+        /// Gets information about an existing move from the table.
+        /// </summary>
+        /// <param name="client">An initialized Amazon DynamoDB client object.</param>
+        /// <param name="newMovie">A Movie object containing information about
+        /// the movie to retrieve.</param>
+        /// <param name="tableName">The name of the table containing the movie.</param>
+        /// <returns>A Dictionary object containing information about the item
+        /// retrieved.</returns>
+        public static async Task<Dictionary<string, AttributeValue>> GetItemAsync(AmazonDynamoDBClient client, Movie newMovie, string tableName)
+        {
+            var key = new Dictionary<string, AttributeValue>
+            {
+                ["title"] = new AttributeValue { S = newMovie.Title },
+                ["year"] = new AttributeValue { N = newMovie.Year.ToString() },
+            };
+
+            var request = new GetItemRequest
+            {
+                Key = key,
+                TableName = tableName,
+            };
+
+            var response = await client.GetItemAsync(request);
+            return response.Item;
+        }
+
+        // snippet-end:[DynamoDB.dotnetv3.GetItem]
+
+        // snippet-start:[DynamoDB.dotnetv3.BatchWriteItem]
+
+        /// <summary>
+        /// Loads the contents of a JSON file into a list of movies to be
+        /// added to the DynamoDB table.
+        /// </summary>
+        /// <param name="movieFileName">The full path to the JSON file.</param>
+        /// <returns>A generic list of movie objects.</returns>
+        public static List<Movie> ImportMovies(string movieFileName)
+        {
+            if (!File.Exists(movieFileName))
+            {
+                return null;
+            }
+
+            using var sr = new StreamReader(movieFileName);
+            string json = sr.ReadToEnd();
+            var allMovies = JsonConvert.DeserializeObject<List<Movie>>(json);
+
+            // Now return the first 250 entries.
+            return allMovies.GetRange(0, 250);
+        }
+
+        /// <summary>
+        /// Writes 250 items to the movie table.
+        /// </summary>
+        /// <param name="client">The initialized DynamoDB client object.</param>
+        /// <param name="movieFileName">A string containing the full path to
+        /// the JSON file containing movie data.</param>
+        /// <param name="tableName">The name of the table. Used for retrieving
+        /// information about the table to get the count of items in the table
+        /// after adding a movies imported from a JSON file.</param>
```

Comment:
a movies --> a movie

Reply:
Actually removed the whole description of tableName. I had removed the parameter but forgot to remove the description.